plapp/grd2obj.py

```diff
@@ -0,0 +1,206 @@
+import logging
+import os
+import time
+
+import numpy as np
+import pygmt as pygmt
+from matplotlib.path import Path
+
+import xarray as xr
+import netCDF4
+import dmsh
+import meshio
+
+from src.astro_trans import sph2cart
+from src.unproject_coord import unproject_stereographic
```

Comment:
What are these? Your local files or another project? They seem to be missing...

Reply:
ops, yes, that's a couple of 'transformations libraries' I copied from another project. Should I include them? Or rather directly include those 2 functions in the file? Either works for me...